yamls/ndm-operator.yaml

```diff
@@ -1,55 +1,3 @@
-# Create the OpenEBS namespace
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: openebs
----
-# Create NDM Service Account
-apiVersion: v1
-kind: ServiceAccount
-metadata:
-  name: openebs-maya-operator
-  namespace: openebs
----
-# Define Role that allows operations on K8s pods/deployments
-# in "openebs" namespace
-kind: ClusterRole
-apiVersion: rbac.authorization.k8s.io/v1
-metadata:
-  namespace: openebs
-  name: openebs-ndm-operator
-rules:
-- apiGroups: ["*"]
-  resources: ["nodes", "pods", "services", "endpoints", "events", "configmaps", "secrets", "jobs"]
-  verbs:
-  - '*'
-- apiGroups: ["apiextensions.k8s.io"]
-  resources: ["customresourcedefinitions"]
-  verbs:
-  - '*'
-- apiGroups:
-  - openebs.io
-  resources:
-  - blockdevices
-  - blockdeviceclaims
-  verbs:
-  - '*'
----
-# Bind the Service Account with the Role Privileges.
-kind: ClusterRoleBinding
-apiVersion: rbac.authorization.k8s.io/v1
-metadata:
-  name: openebs-ndm-operator
-  namespace: openebs
-subjects:
-- kind: ServiceAccount
-  name: openebs-maya-operator
-  namespace: openebs
-roleRef:
-  kind: ClusterRole
-  name: openebs-ndm-operator
-  apiGroup: rbac.authorization.k8s.io
----
```

Comment:
Why are we removing these?

Reply:
This file is used with [this script](https://github.com/openebs/cstor-operators/blob/develop/build/generate-manifest.sh#L35) to generate manifest yamls in openebs/cstor-operators repo. This need not have RBAC and Namespace objects. They were initially not a part of this file. Ref: https://github.com/openebs/charts/pull/282. They were added in subsequent PRs which were meant to update this file.

Reply:
Okay.